lib/mocaccino.js

```diff
@@ -139,7 +138,9 @@ module.exports = function (b, opts) {
         var self = this;
         listener.then(function (err, res) {
           if (!err) {
-            setupSource = (res.mocha || '') + res.setup;
+            setupSource = (opts.node ? '' : 'require(\'brout\');\n')
```

Comment:
Mocha 6.2.0 starts adding behavior to `console.log` https://github.com/mochajs/mocha/pull/3725 so it's now needed to require `brout` before requiring `mocha`. I'm not sure if this ad-hoc string doing it here for us is the most elegant way to do it, but then I also didn't want to add a single line file. Any opinions?

Reply:
My opinion is that any hack that get's us there is fine 😆